include/zmq.h

```diff
@@ -639,17 +654,32 @@ typedef struct zmq_poller_event_t
 #else
     int fd;
 #endif
+    void *zid;
```

Comment:
if you move these 2 at the bottom we avoid an ABI breakage for users of DRAFT APIs

Reply:
NVM I'm talking rubbish, if it's on the stack it's going to break anyway